server/app.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI
+from server.routes.search_result import router as SearchResultRouter
```

Comment:
Please add one blank line between two tdifferent types of import. PEP 8 https://peps.python.org/pep-0008/#imports

Reply:
Got it